### ISSUE TITLE
Sidebar filter depth

### ISSUE THREAD

**Issue by isthisnagee** [uoftweb/CoursePages#10]
The sidebar filter needs to be able to show sub-items a a result. Something like this

Consider this list of elements
```
- about
- homeworks
- tests
- assignments
  - assignment 1
  - assignment 2
  - assignment 3
- discussion
```

Currently, if I search for `assignments` in the filter bar, I'll get back this list

```
search term: assignments
---
- assignments
  - assignment 1
  - assignment 2
  - assignment 3
```

Which is great! However, searching for `assignment 1` will give back the original list (no search found defaults to the original list). It would be cool if we could make it so we get something like this:

```
search term: assignment 1
---
result:
- assignments
  - assignment 1
```

**Comment by isthisnagee** [comment]
fixed in #15

**Comment by isthisnagee** [comment]
fixed in #15
